src/components/OwnernameInput/OwnerNameInput.tsx

```diff
@@ -0,0 +1,71 @@
+import Input from '../common/Input/Input';
+import Field from '../layout/Field/Field';
+
+import useAddCardInput from '../../hooks/useAddCardInput';
+
+import { ADD_CARD_FORM_FIELDS, ERRORS } from '../../constants/messages';
+import { isCharacter } from '../../domain/validators';
+
+const { OWNER_NAME } = ADD_CARD_FORM_FIELDS;
+
+interface OwnerNameInputProps {
+  setCardData: (key: keyof CardInfo, newData: CardInfo[keyof CardInfo]) => void;
+}
+
+function OwnerNameInput({ setCardData }: OwnerNameInputProps) {
+  const validateInputOnChange = ({
+    value,
+  }: {
+    name?: string;
+    value: string;
+  }) => {
+    if (value !== '' && !isCharacter(value)) {
+      return { isValid: false, errorMsg: ERRORS.isNotAlphabet };
+    }
+    return { isValid: true, errorMsg: '' };
+  };
+
+  const processData = () => {
+    setCardData('ownerName', Object.values(ownerName));
+  };
+
+  const {
+    values: ownerName,
+    errMsg,
+    isError,
+    onChange,
+    onBlur,
+  } = useAddCardInput<OwnerName>({
+    initialValues: {
+      ownerName: '',
+    },
+    initialErrors: {
+      ownerName: false,
+    },
+    validateInputOnChange,
+    processData,
+  });
+
+  return (
+    <Field
+      title={OWNER_NAME.title}
+      labelText={OWNER_NAME.labelText}
+      errMsg={errMsg}
+    >
+      {Object.keys(ownerName).map((name) => (
+        <Input
+          key={name}
+          name={name as keyof OwnerName}
+          placeholder={OWNER_NAME.placeholder}
+          value={ownerName[name as keyof OwnerName]}
+          isError={isError[name as keyof OwnerName]}
+          isRequired={true}
```

Comment:
이렇게 간결하게 써도 좋을 것 같습니다 :) 
attribute랑 완전히 동일하게 `required`로 네이밍해도 좋을 것 같은데, 프로젝트 전체에 `is-*` 컨벤션을 적용해주시기 위해 이렇게 네이밍해주신 것으로 이해했어요 👍

```suggestion
          isRequired
```

Reply:
제안해주신 방법이 가능한지 몰랐네요! 알려주셔서 감사합니다.😊